src/NHapi.Base/Model/AbstractGroupItem.cs

```diff
@@ -1,91 +1,79 @@
-using System;
-using System.Collections.Generic;
-using System.Text;
-using NHapi.Base.Model;
-
 namespace NHapi.Base.Model
 {
-	/// <summary>
-	/// Abstract group item
-	/// </summary>
-	public class AbstractGroupItem
-	{
-		private string _name;
-		private List<IStructure> _structures = new List<IStructure>();
-		private bool _isRequired = false;
-		private bool _isRepeating = false;
-		private Type _class;
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Abstract group item.
+    /// </summary>
+    public class AbstractGroupItem
+    {
+        private List<IStructure> structures = new List<IStructure>();
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="required"></param>
+        /// <param name="repeating"></param>
+        /// <param name="classType"></param>
+        public AbstractGroupItem(string name, bool required, bool repeating, Type classType)
+        {
+            Name = name;
+            IsRequired = required;
+            IsRepeating = repeating;
+            ClassType = classType;
+        }
 
-		/// <summary>
-		/// Constructor
-		/// </summary>
-		/// <param name="name"></param>
-		/// <param name="required"></param>
-		/// <param name="repeating"></param>
-		/// <param name="classType"></param>
-		public AbstractGroupItem(string name, bool required, bool repeating, Type classType)
-		{
-			_name = name;
-			_isRequired = required;
-			_isRepeating = repeating;
-			_class = classType;
-		}
+        /// <summary>
+        /// Name of the item.
+        /// </summary>
+        public string Name { get; }
 
-		/// <summary>
-		/// Name of the item
-		/// </summary>
-		public string Name
-		{
-			get { return _name; }
-		}
+        /// <summary>
+        /// The type of class.
+        /// </summary>
+        public Type ClassType { get; }
 
-		/// <summary>
-		/// The type of class
-		/// </summary>
-		public Type ClassType
-		{
-			get { return _class; }
-		}
+        /// <summary>
+        /// Is item repeating.
+        /// </summary>
+        public bool IsRepeating { get; }
 
-		/// <summary>
-		/// Is item repeating
-		/// </summary>
-		public bool IsRepeating
-		{
-			get { return _isRepeating; }
-		}
+        /// <summary>
+        /// Is item required.
+        /// </summary>
+        public bool IsRequired { get; }
 
-		/// <summary>
-		/// Is item required
-		/// </summary>
-		public bool IsRequired
-		{
-			get { return _isRequired; }
-		}
+        /// <summary>
+        /// The structures of the group item.
+        /// </summary>
+        public List<IStructure> Structures

```

Comment:
Can we get rid of the private field and just use the property?

Reply:
done